models/user.py

```diff
@@ -21,6 +21,21 @@ def make(self):
         profile['institution'] = {'name': self.institution_name, 'photo_url': self.institution_photo_url}
         return profile
 
+    @staticmethod
+    def is_valid_profile(profiles, institution_key):
+        """Verify the user profile."""
+        for profile in profiles:
+            if profile.institution_key == institution_key:
+                return InstitutionProfile.is_valid_profile_fields(profile)
+        return False
+
+    @staticmethod
+    def is_valid_profile_fields(profile):
```

Comment:
pq essa função? no return da func acima vc pode colocar `return profile.office is not None`